docs/reference/contracts/deployment-addresses/zkevm.md

```diff
@@ -70,8 +69,8 @@ These deployments were in use at some point, and may still be in active operatio
 #### If you can only find the contract you are looking for in the deprecated section and it is not an old pool, try checking the deployments tasks to find it or ask in the Discord before using a deprecated contract.
 
     
-No deprecated contracts found
-
+| Contract   | Address   | Deployment   |
+|------------|-----------|--------------|
```

Comment:
Ah, it looks like if you don't find anything the code doesn't throw an exception now, so you get an empty table.
If you want to print `No deprecated contracts found` as before maybe that case can be handled differently by checking whether the returned `dict` is empty or not.